apps/bletiny/src/gatt_svr.c

```diff
@@ -366,7 +366,7 @@ static int
 gatt_svr_access_test(uint16_t conn_handle, uint16_t attr_handle,
                      struct ble_gatt_access_ctxt *ctxt,
                      void *arg)
-{
+
```

Comment:
This looks more like a GH quirk. This was a complicated merge but the resulting file looks ok to me.